product_docs/docs/biganimal/release/overview/deployment_options/postgres_admin_user.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: BigAnimal's cloud account ba_admin_user
+---
+
+Learn about what operations the pg_ba_admin user can perform and what Postgres built-in roles it has.
+
+## What pg_ba_admin can do
+The pg_ba_admin user can perform the following operations:
+- Create roles
+- Create databases
+- Read and write to all user objects (tables, views, and so on)
+- Perform all monitoring functions on the Postgres instance (it has the pg_monitor role)
+- Install supported EDB extensions (see [Supported Postgres extensions and tools](biganimal/latest/overview/extensions_tools/))
+
+## What pg_ba_admin can't do
+Using any Postgres distribution, the pg_ba_admin user can't perform the following operations that a Postgres superuser can:
```

Comment:
The overloading of "distribution" here is weird. Can we just remove the part of the sentence before the comma?